client/question/services/CurrentQuestionService.js

```diff
@@ -39,6 +39,8 @@ tie.factory('CurrentQuestionService', [
             cachedQuestion = question;
             serviceIsInitialized = true;
             successCallback();
+          } else if (questionId === DEFAULT_QUESTION_ID) {
+            successCallback();
```

Comment:
One last check: should this be calling successCallback()? It seems like a clear error case.

Reply:
yeah -- it still needs to go through the motions of initializing the LearnerViewDirective, otherwise it doesn't call the redirect.

Reply:
OK. Seems misleading though; add comment in the code to explain please?